node/network/availability-distribution/src/tests/mock.rs

```diff
@@ -114,7 +116,11 @@ impl TestCandidateBuilder {
 				pov_hash: self.pov_hash,
 				relay_parent: self.relay_parent,
 				erasure_root: self.erasure_root,
-				..Default::default()
+				collator: CollatorId::from(sr25519::Public::from_raw([42; 32])),
+				persisted_validation_data_hash: dummy_hash(),
+				signature: sr25519::Signature::from_raw([42; 64]).into(),
```

Comment:
Doesn't get its own helper?

Reply:
A residue, `dummy_collator_signature()` exists and should be used, ty!